### PR TITLE
Update punjabi strings

### PR DESCRIPTION
**Pull request by schima**
updated all EN strings into punjabi. Please review.

### CODE REVIEW

**Comment by schima** [comment]
recheck

File: i18n/intl_messages_pa.arb [hunk 1]
```diff
@@ -13,103 +13,103 @@
...
   },
-  "settings": "Settings",
+  "settings": "ਸੈਟਿੰਗਜ਼",
   "@settings": {
     "description": "Settings",
     "type": "text",
     "placeholders": {}
   },
-  "ordersTitle": "Orders",
+  "ordersTitle": "ਆਦੇਸ਼",
```
Comment (harsimranmaan) [inline i18n/intl_messages_pa.arb:52]: Order like "Aardar". not really a command. Cannot think of something better

File: i18n/intl_messages_pa.arb [hunk 2]
```diff
@@ -119,7 +119,7 @@
       }
     }
   },
-  "emailSent": "An Email has been sent to {emailId}. Please check your inbox.",
+  "emailSent": "emailId} ਨੂੰ ਈਮੇਲ ਭੇਜੀ ਗਈ ਹੈ. ਕਿਰਪਾ ਕਰਕੇ ਆਪਣੇ ਇਨਬਾਕਸ ਦੀ ਜਾਂਚ ਕਰੋ।",
```
Comment (harsimranmaan) [inline i18n/intl_messages_pa.arb:122]:
```suggestion
  "emailSent": "{emailId} ਨੂੰ ਈਮੇਲ ਭੇਜੀ ਗਈ ਹੈ. ਕਿਰਪਾ ਕਰਕੇ ਆਪਣੇ ਇਨਬਾਕਸ ਦੀ ਜਾਂਚ ਕਰੋ।",
```

**Comment by harsimranmaan** [comment]
ਧੰਨਵਾਦ

**Comment by schima** [comment]
recheck

**Review by harsimranmaan — APPROVED** [review]

**Pull request merged**